packages/grid/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -371,6 +371,13 @@ export interface DataGridPropsWithDefaultValues {
    * @default '\t'
    */
   clipboardCopyCellDelimiter: string;
+  /**
+   * The milliseconds delay to wait after measuring the row height before recalculating row positions.
+   * Setting it to a lower value could be useful when using dynamic row height,
+   * but might reduce performance when displaying a large number of rows.
+   * @default 166
+   */
+  rowPositionsDebounceMs: number;
```

Comment:
I'm not 100% sure about the naming, feel free to suggest a better one if you have ideas

Reply:
Also not sure but I don't have anything better to propose at the moment.